### ISSUE TITLE
May I  ask you someting about this code?

### ISSUE THREAD

**Issue by monki-ma** [hbahadirsahin/gmvae#1]
Hello author， First of all, I would like to pay tribute to you for such a good work. I would like to ask you what is **vali_ds** and how **split_vali** works , thank you very much!

**Comment by monki-ma** [comment]
and I'm sorry for someting.....it's something

**Comment by hbahadirsahin** [comment]
vali_ds is the validation set (validation data split).

For splitting a "not-mnist" dataset into 3 splits (train, test, validation): (1) I split the dataset to train and test sets at first, (2) If `split_vali=True`,  half of the test set is initialized as validation set, (3) Otherwise, you don't create your validation set and continue with train and test sets only.

**Comment by monki-ma** [comment]
thank you for your early reply! and how to use mnist with label to let the acc work?

**Comment by hbahadirsahin** [comment]
If you referring how to use this code for classification, you need to make changes. You need to change the loss function from MSE to a type of categorical loss (cross entropy/binary cross entropy). Probably, you will need to write additional code for evaluation.

**Comment by monki-ma** [comment]
ok I will try to combine your work and my own code ,thank you for all your reply! Good luck
